src/views/Tune.vue

```diff
@@ -12,7 +15,7 @@
       md="4"
     >
       <bed-mesh-controls
-        v-if="supportsBedMesh && klippyReady"
+        v-if="supportsBedMesh"
         class="mb-2 mb-sm-4"
       />
       <end-stops-card class="mb-2 mb-sm-4" />
```

Comment:
The query button should probably be disabled when Klippy isn't ready, too

Reply:
No need as we are hiding all cards (hence why I removed it from here)